src/cpu/aarch64/brgemm/brgemm_types.hpp

```diff
@@ -191,6 +191,8 @@ struct brgemm_t {
     int LDB = 0;
     int LDC = 0;
     int LDD = 0;
+
+    int M, K, N;
```

Comment:
Are those used anywhere? I see them set in brgemm_matmul.cpp, but it seems only M,N,K from brgemm_matmul_conf or matmul_params are used all over (and not from brgemm_t).

Reply:
continuing discussion [here](https://github.com/oneapi-src/oneDNN/pull/2103/files#r1772962084), thanks.